quixstreams/sinks/community/file/__init__.py

```diff
@@ -0,0 +1,141 @@
+import logging
+import re
+from pathlib import Path
+from typing import Literal, Union
+
+from quixstreams.sinks import BatchingSink, SinkBatch
+
+from .formats import Format, JSONFormat, ParquetFormat
+
+__all__ = [
+    "FileSink",
+    "InvalidFormatError",
+    "JSONFormat",
+    "ParquetFormat",
+]
+
+logger = logging.getLogger(__name__)
+
+FormatName = Literal["json", "parquet"]
+
+_FORMATS: dict[FormatName, Format] = {
+    "json": JSONFormat(),
+    "parquet": ParquetFormat(),
+}
+
+_UNSAFE_CHARACTERS_REGEX = re.compile(r"[^a-zA-Z0-9 ._]")
+
+
+class InvalidFormatError(Exception):
+    """
+    Raised when the format is specified incorrectly.
+    """
+
+
+class FileSink(BatchingSink):
```

Comment:
Is it normal to keep class in init file? I would not expect it here.

Reply:
There is technically nothing wrong with it, but I will move it to file.py and import the class in __init__.py